libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/changeSharingHandler.ts

```diff
@@ -52,7 +52,9 @@ function* createDashboardSaveSharingContext(cmd: ChangeSharing): SagaIterator<Da
     };
     const dashboardToSave: IDashboardDefinition = {
         ...dashboardFromState,
-        ...newSharingProperties,
+        shareStatus: newSharingProperties.shareStatus,
+        isLocked: newSharingProperties.isLocked,
+        isUnderStrictControl: newSharingProperties.isUnderStrictControl,
```

Comment:
I can see that this omits `grantees..`. Is this intentional?

Reply:
Definitely. This code is building dashboard to save so grantees should not be there. It was failing before the change as it tried to save the dashboard (which was in parent workspace) to the child workspace. For some reason, when the original and new dashboards are the same, it skips the saving (updateDashboard)